### DISCUSSION TITLE
Change what page is "next" / "previous"?

### DISCUSSION THREAD

**Discussion by bannmann** [squidfunk/mkdocs-material#5246] [Q&A]
Here's my current navigation structure:

- Home
    - Introduction❶
    - Privacy Policy❷
    - Imprint❸
- User Guide
    - Getting started❹
    - Examples❺
- References
    - Data mapping details❻
    - Conditions❼
    - Transformers❽
    - Properties❾

I really like the next/previous links in the footer and would like to keep them. However, I envision them being more suited as a "guided tour", so I would like to exclude pages ❷ and ❸ as they are not part of the recommended reading order.

While I can disable the footer on those pages, they still are part of the "tour", i.e. they appear on "neighbor pages" as next/previous.

So instead of going like this:
❶⯈❷⯈❸⯈❹⯈❺⯈❻⯈❼⯈❽⯈❾
❶⯇❷⯇❸⯇❹⯇❺⯇❻⯇❼⯇❽⯇❾

I would like the next/previous links to go like this:
❶⯈❹⯈❺⯈❻⯈❼⯈❽⯈❾
❶⯇❹⯇❺⯇❻⯇❼⯇❽⯇❾

Is this possible by overriding some partials/blocks? If so, what's the best approach? I don't need a full copy-paste solution and am happy to figure out the details myself. I'd just like some reassurance and guidance before entering the rabbit hole.

**Comment by squidfunk** [comment]
This was already requested several times. The solution is to reorder your documentation, i.e., `nav` section. If we would allow to set this via front matter, you would need to change this for every page, because it would be local to each Markdown file. We wouldn't consider this a good idea since it's not really maintainable.
